vendor/composer/ClassLoader.php

```diff
@@ -0,0 +1,445 @@
+<?php
+
```

Comment:
This shouldn't be here!

Reply:
This is third party so haven't touched this!